packages/blockchain-extension/extension/commands/importSmartContractPackageCommand.ts

```diff
@@ -31,7 +31,7 @@ export async function importSmartContractPackageCommand(): Promise<void> {
         canSelectFolders: false,
         canSelectMany: false,
         openLabel: 'Select',
-        filters: { Packages: ['cds'] }
+        filters: { Packages: ['tar.gz', 'tgz'] }
```

Comment:
If you haven't already, it's worth checking that what we don't expect the file extension to only be `.tar.gz` now that we can provide a `.tar.gz` or `.tgz`.

Reply:
Will I be able to import both `contract@0.0.1.tar.gz` and `contract@0.0.1.tgz`?

Reply:
Also, do we validate the name of the imported package? Does it even matter - do we use the package name for anything other than the tree item label?

Reply:
the package files in the tests have a mixture of tar.gz and .tgz

Reply:
yeah you can import either, the name of the imported package doesn't matter. We will eventually use it for the definition when you approve